src/SablierMerkleLL.sol

```diff
@@ -89,29 +89,27 @@ contract SablierMerkleLL is
         // Load schedule from storage into memory.
         MerkleLL.Schedule memory schedule = _schedule;
 
-        // Calculate the timestamps for the stream.
+        // Calculate the timestamps. Zero is used a sentinel value for `block.timestamp`.
```

Comment:
> Zero is used a sentinel value for `block.timestamp`

This is more appropriate when setting value for `schedule` and not when calculating value for `timestamps`. The latter does not use 0 value.

`Schedule` struct in Datatype has already mentioned the significance of 0. For clarity to users, we can also mention it into the natspecs of `SablierMerkleFactoryLL.createMerkleLL` 

If we want to mention it here as well, it should be over `if` statement as `Set startime to block.timestamps if schedule start time is 0`

Reply:
I have a preference for keeping the 'sentinel value' terminology. WDYT about this?

```solidity
// Zero is used a sentinel value for `block.timestamp`.
if (schedule.startTime == 0) {
```